recipe/meta.yaml

```diff
@@ -40,6 +45,8 @@ test:
   commands:
     - flake8 --ignore=D203,W503,E203 src/flake8 tests/integration tests/unit setup.py  # [unix]
     - flake8 --ignore=D203,W503,E203 src\\flake8 tests\\integration tests\\unit setup.py  # [win]
+    # the pip test does not work at this time
+    # - pip check
```

Comment:
We can return pip check as the issue with  `pycodestyle >=2.8.0,<2.9.0` and `pyflakes >=2.4.0,<2.5.0` was resolved